app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt

```diff
@@ -0,0 +1,51 @@
+package org.fossasia.openevent.general
+
+import android.os.Bundle
+import android.support.design.widget.BottomNavigationView
+import android.support.v4.app.Fragment
+import android.support.v7.app.AppCompatActivity
+import android.view.Menu
+import kotlinx.android.synthetic.main.activity_auth.*
+import org.fossasia.openevent.general.auth.LoginFragment
+import org.fossasia.openevent.general.auth.ProfileFragment
+import org.fossasia.openevent.general.auth.SignUpFragment
+import org.fossasia.openevent.general.event.EventsFragment
+
+class AuthActivity : AppCompatActivity() {
+
+    private val listener = BottomNavigationView.OnNavigationItemSelectedListener { item ->
+        val fragment: Fragment
```

Comment:
Why declared here

Reply:
Still here

Reply:
Even in Main activity its declared in the listener. Where should it be declared ?

Reply:
Should it be declared inside the case ?

Reply:
Everything should be declared as locally as possible

Reply:
Sure : Will keep that in mind from now